app/main.py

```diff
@@ -2,0 +1,47 @@
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            prev_appetite = self.appetite
+            self.appetite = 0
+            self.is_hungry = False
+            return prev_appetite
+        return 0
+
+
```

Comment:
There is an unnecessary blank line here. While not critical, it might be better to remove it for cleaner code formatting.